### DISCUSSION TITLE
请问下，skynet中如何做mysql事务

### DISCUSSION THREAD

**Discussion by FashGek** [cloudwu/skynet#1422] [Q&A]
有需要对多个表做事务处理，保持数据的一致性，这个 在skynet的Mysql库中支持吗

**Comment by cloudfreexiao** [comment]
https://github.com/cloudwu/skynet/blob/master/test/testmysql.lua#L80-L118

**Reply by FashGek** [reply]
关于prepare/execute，我一开始也认为这个是用来做事务的，但是我没有找到类似提交、回滚的用法。还是说不需要回滚，
local stmt1 = db:preapre("")
// 成功
 db:execute(stmt1)

local stmt2 = db:prepare("")
local res = db:execute(stmt2)

// stmt2 出错，这时如何回滚stmt1的执行结果呢
